### PR TITLE
New event for January

### PR DESCRIPTION
**Pull request by cryptotwilight**
This event will introduce attendees to the basics of blockchain and decentralized storage as provided on IPFS. It will be your opportunity ask questions and gain answers on the foundations of the Web3 Evolution. This event will be presented in both German and English. We will also have mini workshops that show you how to use Web3 to apply for roles on JobCrypt so if you're early you might get some free crypto. If not fear not we have more swag to give away through the course of the evening, so do join us.

### CODE REVIEW

**Review by pandeyanujk — APPROVED** [review]
Thanks for submitting!

File: content/events/jobcrypt-web3-&-blockchain-talks-munich-germany.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+eventName: JobCrypt Web3 & Blockchain Talks Munich Germany, Sponsored by the Filecoin Orbit Programme
+location: "Munich, Germany"
+website: [https://www.jobcrypt.com/pages/promotions/munich/2023-01-25-In-Person-Talk.html](https://www.jobcrypt.com/pages/promotions/munich/2023-01-25-In-Person-Talk.html)
```
Comment (pandeyanujk) [inline content/events/jobcrypt-web3-&-blockchain-talks-munich-germany.md:4]: @cryptotwilight can you please change this from markdown to just URL "https://..."

Thanks!

**Review by pandeyanujk — CHANGES_REQUESTED** [review]
See - https://github.com/pathfindertools/pl-network-events/pull/17/files#r1070166833

**Review by pandeyanujk — APPROVED** [review]

**Comment by tbenbow** [comment]
Thanks for this @cryptotwilight - this PR has been merged.

**Pull request merged**